EndlessClient/Controllers/LoginController.cs

```diff
@@ -229,6 +230,15 @@ private async Task<bool> SafeGetFile(Func<Task> operation)
             return await op.Invoke();
         }
 
+        private void ClearChat()
```

Comment:
This workaround is fine, but add a todo note here to properly fix the `IResettable` issue with the repository

Reply:
Added TODO comment :)